ledger/catchpointtracker_test.go

```diff
@@ -377,9 +377,6 @@ func BenchmarkLargeCatchpointDataWriting(b *testing.B) {
 func TestReproducibleCatchpointLabels(t *testing.T) {
 	partitiontest.PartitionTest(t)
 
-	if runtime.GOARCH == "arm" || runtime.GOARCH == "arm64" {
```

Comment:
please return, this test is quite heavy

Reply:
sorry, removing them without noticing the heavy test.